doc/ref_model/chapters/glossary.md

```diff
@@ -98,3 +98,5 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Virtualised Infrastructure Manager (VIM)**: responsible for controlling and managing the **NFVI** compute, storage and network resources.
 - **NFV Orchestrator (NFVO)**: manages the VNF lifecycle and **NFVI** resources (supported by the **VIM**) to ensure an optimised allocation of the necessary resources and connectivity.
+- **Platform**: all hardware and software that serves as a foundation or base to host VNFs and CNFs. 
+>_*Note:*_ For CNTT,  this includes the physical infrastructure, Operating Systems, virtualization/containerization software and other orchestration, security, monitoring/logging and life-cycle management software.
```

Comment:
I have no problems with this definition as written, but as with all the others I'd rather link to and/or work from an existing well-defined and industry wide definition.  e.g. the ITU defines "platform" as:

> A cloud capabilities type in which the cloud service customer can deploy, manage and run customer-created or customer-acquired applications using one or more programming languages and one or more execution environments supported by the cloud service provider.

We could use `cloud service user` instead of `cloud service customer` as we already have that definition in the Glossary but like I say, depends whether we want to create our own definitions for things or not.

Reply:
@tomkivlin Thanks. I am OK with the ITU definition with your suggested change. May I suggest that we include a Note so that it is clear that the plaltform includes hardware and software.

Reply:
@pgoyal01 can you please add the note ad agreed above in a new commit?

Reply:
Done